backend/Spring Boot/springboot_starterproject/src/main/java/daiict/slop/springboot_starterproject/controller/HealthController.java

```diff
@@ -0,0 +1,23 @@
+package daiict.slop.springboot_starterproject.controller;
+
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+
+/**
+ * created by Ekta on 01/10/2020
+ */
+
+@RestController
+public class HealthController {
+
+    /**
+     * Api to check project is working correctly.
```

Comment:
```suggestion
     * Health Check API
```
This intro will work